python_bindings/tests/end-to-python-end-test/other_two_player_game_test.py

```diff
@@ -0,0 +1,636 @@
+# flake8: noqa ascci drawing makes a lot errors
+# mypy: ignore-errors
+""""""
+
+from carcassonne_engine._bindings.elements import MeepleType
+from carcassonne_engine._bindings.side import Side
+from carcassonne_engine.placed_tile import Position
+
+"""
+ diagonal edges represent cities, dots fields, straight lines roads.
+ Player meeples will be represented as !@ signs ( you know, writing number but with shift!) 1->!, 2->@
+ Final board: (each tile is represented by 5x5 ascii signs, at the center is the turn number in hex :/)
+
+|				  0	   1    2    3
+|
+|
+|               |   |...............
+|               .\ /............[ ].
+|1              ..2....4----9---[A].
+|               ./ \...|.../ \..[ ].
```

Comment:
Because of mixed use of tabs and spaces here, the '0 1 2 3' don't align with the board below

same thing in all subsequent images

Reply:
fixed